minio/minio
# Add grafana json file for replication metrics

## Description
Add a new file names `minio-replication.json` which displays the replication metrics in Grafana dashboard.

## Motivation and Context


## How to test this PR?

 Refer https://github.com/minio/minio/blob/master/docs/metrics/prometheus/grafana/README.md
 Use `minio-replication.json`  in the grafana dashboard to see the replication metrics 

## Types of changes

## Checklist:
